src/main/java/org/kainos/ea/client/BandDoesNotExistException.java

```diff
@@ -0,0 +1,9 @@
+package org.kainos.ea.client;
+
+public class BandDoesNotExistException extends Throwable {
+    @Override
+    public String getMessage() {
+        return
+                "Band selected does not exist";
```

Comment:
This is very weird spacing! haha Why use two lines when you could use one?

Reply:
Believe it was a linter error I was trying to fix and it messed with the formatting